provider/azure/storage.go

```diff
@@ -89,22 +103,21 @@ func (e *azureStorageProvider) Dynamic() bool {
 	return true
 }
 
+// DefaultPools is defined on the Provider interface.
```

Comment:
should this be an "implements" boilerplate? (or my preference, "Foo is part of the Bar interface", which remains correct and minimally misleading however large the Bar interface may be)